packages/core/src/base/Time.ts

```diff
@@ -1,15 +1,34 @@
+import { Vector4 } from "@oasis-engine/math";
+import { ShaderData } from "../shader/ShaderData";
+import { ShaderProperty } from "../shader/ShaderProperty";
+
 /**
- * Tools for calculating the time per frame.
+ * Provide time related information.
  */
 export class Time {
-  /** @internal */
-  _frameCount: number = 0;
+  private static _elapsedTimeProperty = ShaderProperty.getByName("oasis_ElapsedTime");
+  private static _deltaTimeProperty = ShaderProperty.getByName("oasis_DeltaTime");
```

Comment:
`oasis_ElapsedTime` is not suitable, it does not indicate the type of the attribute, and it is not convenient for the engine to rename